src/components/layout/CommandPaletteModal.tsx

```diff
@@ -65,6 +75,11 @@ const CommandPaletteModal = (props: Props) => {
               background:
                 theme === 'dark' ? 'rgba(0,0,0,0.5)' : 'rgba(0,0,0,0.2)',
             }}
+            onClick={e => {
+              if (e.target === e.currentTarget) {
+                onClose();
+              }
+            }}
```

Comment:
Why do we need this? I think we can already close the modal after clicking the overlay